proposals/stage-2/CPC_CHARTER/CPC-CHARTER.md

```diff
@@ -247,8 +247,8 @@ processes for filling this seat will be as follows:
 
 * As a constellation member of the OpenJS Foundation, the Node.js project
   will elect one Board representative from either the TSC or
-  Community Committee. The representative will be selected through a vote
-  of the members of the Node.js organization. 
+  Community Committee. The representative will be selected according to a process
+defined by the TSC and CommComm. 
```

Comment:
Nit: This and line 249 are the only instances of `TSC` anywhere in the doc. It never spells it out as `Technical Steering Committee`.  I think it would be better to use the full name and ditch the acronym unless the term starts getting used more in the doc. Even then, though, it should be spelled out the first time at least.

Reply:
Also nit: This is the first and only use of `CommComm` and it likewise is not explained. You sort of have to infer that it's the aforementioned Community Committee. Not a huge deal, but probably slightly better to just say `Community Committee`.

Reply:
maybe s/defined by the TSC and CommComm./defined by the leadership of the Node.js project

Reply:
Saying because I'd like to keep discussing, while the TSC and CommComm exist, using a more general "leadership of the Node.js project" seems, well, more general than we need at this point.

Reply:
I think this is ok right now. Can you check?

Reply:
Looks like indentation was accidentally removed on one line, but the actual text LGTM now.

Reply:
```suggestion
  defined by the TSC and CommComm. 
```